packages/web-components/fast-ssr/tsconfig.json

```diff
@@ -4,7 +4,6 @@
     "target": "ES2015",
     "module": "ES2015",
     "moduleResolution": "NodeNext",
-    "importHelpers": true,
```

Comment:
Do we not need tslib/importHelpers here anymore?

Reply:
Added them back in and upgraded typescript for this package.